_posts/blog/2021-01-31-Welcome-Alfredo.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "The Zcash Foundation Welcomes A New Core Engineer"
+excerpt: "Welcome, Alfredo!"
+categories: blog
+tags: [team]
+date: 2021-01-31
+author: antonie
+---
+
+The Zcash Foundation is very excited to welcome Alfredo Garcia as our Core Engineer. 
+
+Alfredo is a self-taught developer who has worked in the blockchain industry for the last 5 years as an active defender and contributor of the open source community. He started working for the Zcash protocol one year ago as a grant recipient from the Zcash Foundation. As an outside contributor Alfredo first worked on zcashd development and eventually focused on zebrad. While working on zebrad, he worked side by side with the Zebra team forming a productive relationship, which led to the Foundation offering him a role as a core engineer.
+Alfredo's main role with the Foundation is to help with the design and development of the zebrad client.
```

Comment:
Do we want to say `Zebra` rather than `zebrad`?

It's easier to see that it's different from `zcashd`, and also more technically accurate.

`Zebra` is a set of libraries used to create `zebrad`, but other projects are already using `zebra-network`, and we have plans to make `zebra-client` using the same libraries.

Reply:
whatever you think is best.

Reply:
Ok I'll do the replacement

```suggestion
Alfredo is a self-taught developer who has worked in the blockchain industry for the last 5 years, as an active defender and contributor to the open source community. He started working for the Zcash protocol one year ago as a grant recipient from the Zcash Foundation. As an outside contributor Alfredo first worked on zcashd development and eventually focused on Zebra. While working on Zebra, he worked side by side with the Zebra team forming a productive relationship, which led to the Foundation offering him a role as a core engineer.

Alfredo's main role with the Foundation is to help with the design and development of Zebra, the Foundation's Zcash protocol implementation.
```